content/messaging/agent-chat-api/changelog/index.mdx

```diff
@@ -17,6 +17,7 @@ This document is the record of all the changes in the **Agent Chat API**, Web an
   - [**chat_deleted**](/messaging/agent-chat-api/v3.4/rtm-reference/#chat_deleted)
   - [**thread_deleted**](/messaging/agent-chat-api/v3.4/rtm-reference/#thread_deleted)
   - [**threads_deleted**](/messaging/agent-chat-api/v3.4/rtm-reference/#threads_deleted)
+  - [Configuration related pushes](/messaging/agent-chat-api/v3.4/rtm-reference/#group_created)
```

Comment:
Shouldn't we list all 3 pushes here?

Reply:
nah, i wanted to have a link to entire section, should be better now